src/styles/index.css

```diff
@@ -0,0 +1,124 @@
+body {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  align-items: center;
+  justify-content: center;
+}
+
+.product-card__image {
+  background-image: url('../images/imac.jpeg');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
+  box-sizing: border-box;
+}
+
+.product-card__content {
+  margin: 40px 16px 16px;
+  font-family: Roboto, sans-serif;
+}
+
+.product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 4px 0 0;
+}
+
+.product-rating {
+  margin: 16px 0 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.product-rating__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.stars {
+  flex-direction: row;
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  background-image: url('../images/star-active.svg');
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-of-type {
+  background-image: url('../images/star.svg');
```

Comment:
Using two different classes to apply a background image for active and inactive stars may not be the most efficient approach. Consider using a single class and modifying the background image with a modifier class or using a pseudo-class if the inactive state is based on user interaction.